.github/workflows/release-publish.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       env:
         GIT_REPO: "ppa:git-core/ppa"
 
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
```

Comment:
Have to be careful here as latest git actions don't pull tags. Should be tested before merge.

Reply:
Tested at fork, works fine.

https://github.com/siemens/fossology/actions/runs/12461672380/job/34781573645?pr=42#step:9:66